packages/analytics-core/src/core-client.ts

```diff
@@ -127,8 +127,8 @@ export class AmplitudeCore implements CoreClient {
 
       return result;
     } catch (e) {
+      this.config.loggerProvider.error(e);
```

Comment:
I see that the Logger interface accepts `...args: any[]` but wonder if this will make implementing custom loggers harder for customers vs simple strings (each logger will need to implement the string conversion itself).

Since this doesn't change the public interface I think this is ok, just something to think about.

Reply:
I think because it's already of `any` type and we never committed to a string parameter, the custom implementation should handle conversion to the preferred type. So i think we should be ok.